OpenMEEG/wrapping/src/test_python.py

```diff
@@ -1,4 +1,20 @@
 #!/usr/bin/env python
+###########################################
+# CAVEAT on using OpenMEEG from python !!
+# Beware that a temporary object has its memory
+# released. So do not work with data provided from
+# an OpenMEEG temporary object.
+# For example, having a symmetric matrix defined as :
+# > M = om.SymMatrix(100)
+# taking as an numpy array the sub-matrix of M might lead to corrupted memory:
+# > mySubMat = om.asarray(M.submat(0,10,0,10))
+# since submat returns a newly created object that is hold by nothing, thus
+# destroyed afterward.
+# Instead do keep an object pointing the newly created submatrix, and then
+# access the numpy array form of it:
+# > subM = M.submat(0,10,0,10)
+# > mySubMat = om.asarray(subM)
+###########################################
```

Comment:
yes ok.
beside that .. we have a failing test on mac (test_python) and travis says green (because we added the check package for sure...)
99% tests passed, 1 tests failed out of 1165

Total Test time (real) =  85.50 sec

The following tests FAILED:

	348 - openmeeg_python (OTHER_FAULT)